docker-databases/db2_11-5/Dockerfile

```diff
@@ -48,19 +48,27 @@ RUN localedef -i de_DE -c -f UTF-8 \
     -A /usr/share/locale/locale.alias de_DE.UTF-8 && echo "LANG=de_DE.UTF-8" > /etc/locale.conf
 RUN echo "export LANG=de_DE.UTF-8" >> /etc/profile
 
+FROM base AS db2-install
+
 COPY ./db.tar.gz /tmp/expc.tar.gz
+COPY ./db2server.rsp /tmp/db2server.rsp
 
 RUN cd /tmp && tar xf expc.tar.gz \
-    && su - db2inst1 -c "/tmp/expc/db2_install -y -b /home/db2inst1/sqllib" \
+    && su - db2inst1 -c "/tmp/server_dec/db2setup -r /tmp/db2server.rsp" \
     && echo '. /home/db2inst1/sqllib/db2profile' >> /home/db2inst1/.bash_profile \
-    && rm -rf /tmp/db2* && rm -rf /tmp/expc* \
+    && rm -rf /tmp/db2* && rm -rf /tmp/expc* && rm -rf /tmp/server_dec \
     && sed -ri  's/(ENABLE_OS_AUTHENTICATION=).*/\1YES/g' /home/db2inst1/sqllib/instance/db2rfe.cfg \
     && sed -ri  's/(RESERVE_REMOTE_CONNECTION=).*/\1YES/g' /home/db2inst1/sqllib/instance/db2rfe.cfg \
     && sed -ri 's/^\*(SVCENAME=db2c_db2inst1)/\1/g' /home/db2inst1/sqllib/instance/db2rfe.cfg \
     && sed -ri 's/^\*(SVCEPORT)=48000/\1=50000/g' /home/db2inst1/sqllib/instance/db2rfe.cfg
 
+RUN echo "0 localhost 0" > /home/db2inst1/sqllib/db2nodes.cfg
```

Comment:
out of curiosity: why do we have to configure this?

Reply:
we configure localhost as DB2 requires a hostname for the instance.